README.md

```diff
@@ -75,7 +75,7 @@ Create an [issue](https://github.com/vmware/versatile-data-kit/issues) or [pull
     2. Joining the [#versatile-data-kit](https://cloud-native.slack.com/archives/C033PSLKCPR) channel.
 - Follow us on [Twitter](https://twitter.com/VDKProject).
 - Subscribe to the [Versatile Data Kit YouTube Channel](https://www.youtube.com/channel/UCasf2Q7X8nF7S4VEmcTHJ0Q).
-- Join our [mailing list](join-versatiledatakit@groups.vmware.com).
+- Join our [dev mailing list](mailto:join-versatiledatakit@groups.vmware.com) (used by developers and maintainers of VDK).
```

Comment:
I suggest:
Join our [development mailing list](mailto:join-versatiledatakit@groups.vmware.com), used by developers and maintainers of VDK.

Reply:
```suggestion
- Join our [development mailing list](mailto:join-versatiledatakit@groups.vmware.com), used by developers and maintainers of VDK
```

Reply:
Thanks, changed.